Packs/NucleonCyber/Integrations/NucleonCyberFeed/NucleonCyberFeed.py

```diff
@@ -0,0 +1,761 @@
+            'rawJSON': raw_data
+        }
+        indicators.append(indicator_obj)
+
+    return indicators
+
+
+"""
+"""
+
+
+def fetch_urls(client: Client, limit: int = -1) \
+        -> List[Dict]:
+    """Retrieves indicators from the feed
+    Args:
+        client (Client): Client object with request
+        limit (int): limit the results
+    Returns:
+        Indicators.
+    """
+    # iterator = client.build_iterator()
+    # params = demisto.params()
+    iterator = client.get_urls(limit)
+
+    indicators = []
+
+    if limit > 0:
+        iterator = iterator[:limit]
+    for item in iterator:
+        value_ = item.get('value')
+        type_ = item.get('type')
+        raw_data = {
+            'value': value_,
+            'type': type_,
+        }
+
+        # Create indicator object for each value.
+        # The object consists of a dictionary with required and optional keys and values, as described blow.
+        for key, value in item.items():
+            raw_data.update({key: value})
+
+        indicator_obj = {
+            # The indicator value.
+            'value': value_,
+            'type': type_,
+            # The indicator type as defined in Cortex XSOAR.
+            # One can use the FeedIndicatorType class under CommonServerPython to populate this field.
+            # 'exp':exp_,
+            # "port":port_,
+            # The name of the service supplying this feed.
+            'service': 'NucleonCyberFeed',
+            # A dictionary that maps values to existing indicator fields defined in Cortex XSOAR.
+            # One can use this section in order to map custom indicator fields previously defined
+            # in Cortex XSOAR to their values.
+            'fields': {
+            },
+            # A dictionary of the raw data returned from the feed source about the indicator.
+            'rawJSON': raw_data
+        }
+        indicators.append(indicator_obj)
+
+    return indicators
+
+
+def get_indicators_command(client: Client,
+                           params: Dict[str, str],
+                           args: Dict[str, str]
+                           ) -> CommandResults:
+    """Wrapper for retrieving indicators from the feed to the war-room.
+    Args:
+        client: Client object with request
+        params: demisto.params()
+        args: demisto.args()
+    Returns:
+        Outputs.
+        IP indicators
+    """
+    limit = int(args.get('limit', '10'))
+    tlp_color = params.get('tlp_color')
+    feed_tags = argToList(params.get('feedTags', ''))
+    indicators = fetch_indicators(client, tlp_color, feed_tags, limit)
+    human_readable = tableToMarkdown('IP indicators from NucleonCyberFeed:', indicators,
+                                     headers=['value', 'type', 'exp'], headerTransform=string_to_table_header, removeNull=True)
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='NucleonCyber.Indicators',
+        outputs_key_field='',
```

Comment:
Does it have a unique key for each data? (seems to me that value is IP which can be counted as unique).
if so. Perhaps adding this would be nice
```suggestion
        outputs_key_field='value',
```